src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -296,54 +296,34 @@ public void ProcessStart_UseShellExecute_OnUnix_ValidVerbs(string verb, bool isV
             }, verb ?? "<null>", isValid.ToString(), options).Dispose();
         }
 
-        [Theory, InlineData("vi")]
-        [PlatformSpecific(TestPlatforms.Linux)]
-        [OuterLoop("Opens program")]
-        public void ProcessStart_OpenFileOnLinux_UsesSpecifiedProgram(string programToOpenWith)
+        [Fact]
+        public void ProcessStart_OnLinux_UsesSpecifiedProgram()
         {
-            if (IsProgramInstalled(programToOpenWith))
-            {
-                string fileToOpen = GetTestFilePath() + ".txt";
-                File.WriteAllText(fileToOpen, $"{nameof(ProcessStart_OpenFileOnLinux_UsesSpecifiedProgram)}");
-                using (var px = Process.Start(programToOpenWith, fileToOpen))
-                {
-                    Assert.Equal(programToOpenWith, px.ProcessName);
-                    px.Kill();
-                    px.WaitForExit();
-                    Assert.True(px.HasExited);
-                }
-            }
-            else
+            string program = "sleep";
+
+            using (var px = Process.Start("sleep", "30"))
             {
-                Console.WriteLine($"Program specified to open file with {programToOpenWith} is not installed on this machine.");
+                Assert.Equal(program, px.ProcessName);
+                px.Kill();
```

Comment:
I realize you're just copying what was already there, but I think the px.Kill should be in a finally block, with the Equal in a try... otherwise, if the Equal fails, the process may live on.